src/generate_atergo.js

```diff
@@ -1,4 +1,46 @@
-his is  a placeholder for a file
-ABC
-asd
-ASDASDF
+const url = 'https://docs.google.com/spreadsheets/d/1enQL0xIPzRCOuRvc9aB-8KUOgookyz4rqzcQX9jzNyU/edit#gid=273599930';
```

Comment:
This url should be in main file, as it is const. (like menu or sth.)

Reply:
done in fork